paperqa/llms.py

```diff
@@ -764,6 +802,378 @@ async def select_tool(
         return await tool_selector(*selection_args, **selection_kwargs)
 
 
+class LLMBatchModel(ABC, BaseModel):
+    model_config = ConfigDict(extra="forbid", arbitrary_types_allowed=True)
+
+    llm_type: str | None = None
```

Comment:
Can you add a `Field(description="...")` what this should be when it's a `str`?

Imo "type" wording makes me think it should be a `class`, but it's actually really more of an `Enum`. Maybe we should have this be an `str | Enum | None`

Reply:
Yeah, it should be `chat` or `completion.` So it's a `str` with the model's "type." 
It is checked on inference time. If `llm_type` is `None`, then a `infer_llm_type` is called. But I added a description explaining the attribute in case the user wants to define it. Also changed the declaration in `LiteLLMModel`